modules/relisten-audio-player/ios/RelistenGaplessAudioPlayer/PlaybackUpdates.swift

```diff
@@ -42,6 +66,8 @@ extension RelistenGaplessAudioPlayer {
             let oldKilobytes = floor(Double(oldDownloadedBytes) / (100 * 1024))
             let newKilobytes = floor(Double(downloadedBytes) / (100 * 1024))
             
+            updateControlCenter();
```

Comment:
we might want to consider gating this behind `sendPlaybackChanged == true` but it might be premature optimization for native -> native call

Reply:
nice call!